tools/codegen/base/cargo-auditable.json

```
@@ -0,0 +1,16 @@
+{
+  "repository": "https://github.com/rust-secure-code/cargo-auditable",
+  "website": "https://github.com/rustsec/rustsec/tree/HEAD/cargo-audit",
+  "license_markdown": "[Apache-2.0](https://github.com/rust-secure-code/cargo-auditable/blob/HEAD/LICENSE-APACHE) OR [MIT](https://github.com/rust-secure-code/cargo-auditable/blob/HEAD/LICENSE-MIT)",
+  "tag_prefix": "v",
+  "rust_crate": "${package}",
+  "asset_name": "${package}-${rust_target}.tar.xz",
+  "bin": "${package}-${rust_target}/${package}${exe}",
+  "platform": {
+    "x86_64_linux_musl": {},
+    "x86_64_macos": {},
+    "x86_64_windows": {
+      "asset_name": "${package}-${rust_target}.zip"
+    }
```

```suggestion
    },
    "aarch64_macos": {}
```

cargo-auditable also provides aarch64 macos binary.